docker/playground/README.md

```diff
@@ -34,6 +34,8 @@ Kyuubi supply some built-in dataset, after Kyuubi started, you can run the follo
 - MinIO: http://localhost:9001
 - PostgreSQL localhost:5432 (username: postgres, password: postgres)
 - Spark UI: http://localhost:4040 (available after Spark application launching by Kyuubi, port may be 4041, 4042... if you launch more than one Spark applications)
+- Prometheus: http://localhost:9090
+- grafana: http://localhost:3000 (username: admin, password: admin)
```

Comment:
grafana => Grafana 

I remember we can disable the authentication in conf

Reply:
I have tried to set anonymous access grafana, and actually here are same config to allow anonymous to create and edit dashboard, but it required login in to save those dashboard.

Reply:
After changing prometheus datasource not as default, anonymous can create those folders and dashboards.

Reply:
do you forget to update this line then?

Reply:
let's keep username and password here, in case developer want to login in.

Reply:
```suggestion
- Grafana: http://localhost:3000 (username: admin, password: admin)
```